### PR TITLE
Adds a survival mre and makes it the standard MRE

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds an MRE that'll last longer than your units combat life expectancy. It has sugar and 3 protein bars and a canteen.

## Why It's Good For The Game
Adds a more flavorful MRE type to the game that's always set. And is nice.

## Changelog
:cl:
add: Added new MRE type and makes it the standard one you get.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -48. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/game/objects/items/marine_gear.dm [hunk 1]
```diff
@@ -221,6 +221,22 @@
...
+	package = TRUE
+	list_reagents = list(/datum/reagent/consumable/nutriment = 8)
+	tastes = list("nutraloafed food" = 7, "cocoa" = 1)
+
+/obj/item/reagent_containers/food/snacks/packaged_probar/attack_self(mob/user as mob)
+	if(package)
+		playsound(src.loc,'sound/effects/pageturn2.ogg', 15, 1)
+		to_chat(user, "<span class='notice'>You pull off the wrapping from the squishy burrito!</span>")
+		package = FALSE
+		icon_state = "yummers"
```
Comment (TiviPlus) [inline code/game/objects/items/marine_gear.dm:238]: nice copypaste

Use an early return and  change burrito

File: code/game/objects/items/reagent_containers/food/drinks.dm [hunk 2]
```diff
@@ -234,6 +234,13 @@
 	center_of_mass = list("x"=17, "y"=8)
 	list_reagents = list(/datum/reagent/water = 51, /datum/reagent/consumable/ethanol/hooch = 9)
 
+/obj/item/reagent_containers/food/drinks/flask/survival
+	name = "\improper TGMC flask"
+	desc = "A metal flask with a decent liquid capacity."
+	icon_state = "flask_tgmc"
+	center_of_mass = list("x"=17, "y"=8)
+	list_reagents = list(/datum/reagent/water = 30)
+
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/food/drinks.dm:243]: why are you copypasting a new flask instead of using the old one

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Why are you making a new MRE?

Just make add it to the pool of existing MREs, no need to change that

**Comment by Pariah919** [comment]
i wanted this mre to hold different things compared to the normal mre and be set. If its just gonna be a normal MRE I don't really see the point of continuing

**Pull request closed without merging**